morphir-decoration-extension/src/editor.ts

```diff
@@ -0,0 +1,288 @@
+import * as vscode from "vscode";
+import { getDecorations, isConfigured, updateDecorations } from "./decoration-manager";
+import { NodeDetail } from "./nodeDefinitions";
+
+export function getWebviewOptions(
+  extensionUri: vscode.Uri
+): vscode.WebviewOptions {
+  return {
+    // Enable javascript in the webview
+    enableScripts: true,
+
+    // And restrict the webview to only loading content from our extension's `media` directory.
+    localResourceRoots: [
+      vscode.Uri.joinPath(extensionUri, "media"),
+      vscode.Uri.joinPath(extensionUri, "../cli/web"),
+      vscode.Uri.joinPath(extensionUri, "../test-integration"),
```

Comment:
Are all these paths required? The comment only mentions the first one.

Reply:
restrict the webview to only loading content from the directory paths added in the localResourcesRoots